README.md

```diff
@@ -46,6 +46,22 @@ Installation
 TinyFSM is a header-only library, no special installation steps are
 needed. Just point your compiler to the "include" directory.
 
+Alternately, you could install the library using cmake. A typical install procedure is:
+```bash
+mkdir build
+cd build
+cmake ..
+sudo cmake --build . --target install
+```
```

Comment:
cmake users should all know haw this works, do you really think this is required? maybe add instructions in header of CMakeLists.txt